src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/NameQuotedValuePairListEncoding.cs

```diff
@@ -11,12 +11,13 @@ internal sealed class NameQuotedValuePairListEncoding : INameValuePairListEncodi
     {
         public IEnumerable<(string Name, string Value)> Parse(string value)
         {
-            Requires.NotNullOrWhiteSpace(value, nameof(value));
-
             foreach (var entry in ReadEntries(value))
             {
                 var (entryKey, entryValue) = SplitEntry(entry);
-                yield return (DecodeCharacters(entryKey), StripQuotes(DecodeCharacters(entryValue)));
+                if (!string.IsNullOrEmpty(entryKey))
```

Comment:
This will throw away a value having an empty key, which is data loss. Eg. `="value"`.